specification/compute/resource-manager/readme.md

```diff
@@ -445,6 +459,7 @@ swagger-to-sdk:
     after_scripts:
       - python ./scripts/multiapi_init_gen.py azure-mgmt-compute
   - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-js

```

Comment:
I am not sure why its showing as "added" even while comparing with Azure repository.. This change in readme.md was simply pulled in from Azure master to forked repository